src/building/suggested.md

```diff
@@ -38,8 +38,9 @@ you can write: <!-- date: 2022-04 --><!-- the date comment is for the edition be
     ],
     "rust-analyzer.procMacro.enable": true,
     "rust-analyzer.cargo.buildScripts.enable": true,
+    "rust-analyzer.cargo.buildScripts.useRustcWrapper": true,
```

Comment:
This seems to be set by default, why set it explicitly here?
(Same for the `enable` options.)

Reply:
Answered above

Reply:
I think it'd be better to not have these lines.
- If people have this explicitly globally disabled, we should respect that.
- For the libs suggestion, we can just have another code block saying "also add these things".